dll/cpl/desk/background.c

```diff
@@ -486,6 +494,18 @@ InitBackgroundDialog(HWND hwndDlg, PDATA pData)
             SendDlgItemMessage(hwndDlg, IDC_PLACEMENT_COMBO, CB_SETCURSEL, PLACEMENT_STRETCH, 0);
             pData->placementSelection = PLACEMENT_STRETCH;
         }
+
+        if (_ttoi(szBuffer) == 6)
```

Comment:
6?
Would be nice to have a define or something for this.

Reply:
Should I create a new pull request for this? I think it doesn't fit here.

Reply:
Imo it would make sense to do it in this pr, maybe @ThFabba has a different idea about that tho?

Reply:
Separate commit in the same PR would make sense. Though I don't feel strongly about this and would be okay seeing it fixed later.

Reply:
Okay, I will do this in this PR.